Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -651,7 +651,9 @@ extension TabTrayController: UIScrollViewAccessibilityDelegate {
         visibleCells = visibleCells.filter { !CGRectIsEmpty(CGRectIntersection($0.frame, bounds)) }
 
         let cells = visibleCells.map { self.collectionView.indexPathForCell($0)! }
-        let indexPaths = cells.sort { $0.section < $1.section || ($0.section == $1.section && $0.row < $1.row) }
+        let indexPaths = cells.sort { (a: NSIndexPath, b: NSIndexPath) -> Bool in
```

Comment:
Yeah see https://bugs.swift.org/browse/SR-1049 - The compiler can't figure out what to infer and gives up. I must admit that I just put the most hints in there as possible and not tried to simplify it back from there. Not sure if that is worth it.